hw2/dz_2.1.py

```diff
@@ -0,0 +1,11 @@
+list_array = [10, 11, 2, 3, 5, 8, 23, 11, 2, 5, 76, 43, 2, 32, 76, 3, 10, 0, 1]
+print(set(list_array))
+list_array_upt = list_array.copy()
+for i in range (0, 3):
+    print(max(list_array_upt))
+    list_array_upt.remove(max(list_array_upt))
```

Comment:
maybe better to sort the list with reverse=True and show the first 3 elements? 
```
a = sorted(list_array, reverse=True)
print(a[:3])
```

Reply:
My first choice was to use the reverse but it didn't work. Now I see what mistake I had in this structure. But why I have to show only the first 3 elements? I had to show the initial array after revers? Is that correct? Thank you!